bugswarm/client/docker.py

```diff
@@ -113,7 +120,7 @@ def _image_exists_locally(image_tag):
 def _image_location(image_tag):
```

Comment:
Let's refactor this to take in the repo and image tag to avoid code duplication.

Reply:
image_location = '{}:{}'.format(DOCKER_HUB_REPO, image_tag)